dev/io.openliberty.webcontainer.servlet.6.0.internal/src/io/openliberty/webcontainer60/srt/SRTServletResponse60.java

```diff
@@ -0,0 +1,176 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.webcontainer60.srt;
+
+import java.util.Locale;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.ibm.websphere.servlet.response.IResponse;
+import com.ibm.ws.webcontainer.webapp.WebAppDispatcherContext;
+import com.ibm.ws.webcontainer40.srt.SRTServletResponse40;
+import com.ibm.wsspi.webcontainer.logging.LoggerFactory;
+
+import io.openliberty.webcontainer60.osgi.srt.SRTConnectionContext60;
+import jakarta.servlet.http.HttpServletResponse;
+
+public class SRTServletResponse60 extends SRTServletResponse40 implements HttpServletResponse {
+
+    protected static final Logger logger = LoggerFactory.getInstance().getLogger("io.openliberty.webcontainer60.srt");
+    private static final String CLASS_NAME = SRTServletResponse60.class.getName();
+
+    public SRTServletResponse60(SRTConnectionContext60 context) {
+        super(context);
+    }
+
+    @Override
+    public void initForNextResponse(IResponse resp) {
+        super.initForNextResponse(resp);
+    }
+
+    @Override
+    public void setCharacterEncoding(String encoding) {
+        if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled() && logger.isLoggable(Level.FINE)) {
```

Comment:
General question for all of these methods is, basically they are copied from the 40 class, using existing messages correct? Then some new updates for Servlet 6.0, correct?

Reply:
yes, it extends the 4.0.